A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AttitudeIndicator.js

```diff
@@ -1348,12 +1348,18 @@ class Jet_PFD_AttitudeIndicator extends HTMLElement {
         this.applyAttributes();
     }
     applyAttributes() {
+        let horizonHeight = 300;
+        let attitudeHeight = 250;
+        let horizonToAttitudeRatio = attitudeHeight / horizonHeight;
+        let deltaY = this.pitch * this.pitchAngleFactor * this.pitchGradFactor;
```

Comment:
These could all be turned into constants since they aren't reassigned.

Reply:
`this.pitch` isn't a constant. I moved the others to above though in the constructor.